.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: "🐛 Bug report"
+about: Report a bug or regression
+title: ''
+labels: "\U0001F41B bug, awaiting triage"
+assignees: ''
+
+---
+
+<!--
+    Before you raise this issue, please check that the issue isn’t already captured in:
+    - an existing Github issue: https://github.com/alphagov/govuk-frontend/issues
+    - our known validation error/warning list: https://github.com/alphagov/govuk-frontend/issues/1280#issuecomment-509588851
+
+    Please fill in as much of the template below as you’re able to. If you're unsure whether the issue already exists or how to fill in the template, open an issue anyway. Our team will help you to complete the rest.
```

Comment:
How would we feel about putting this section in bold in each of the templates? I'd be really keen to emphasise the points here, especially during the trial phase when we don't know whether  the issue templates could discourage some users from raising issues. Anything that could potentially lower that barrier would be good.

Reply:
@hannalaakso Do you mean moving this out of a comment within the template, into bold text that's actually rendered when the issue is previewed/created?

Reply:
Ignore my comment about bolding (that won't work, will it 😂 ), I was a bit tired when I wrote this. What I meant was that if there's a way of making this section more prominent that would be great. I wonder if we could consider swapping the two paragraphs in the comment as I'd personally much rather that the person raising the issue saw the first paragraph rather than the second one.

Reply:
Haha, no worries! That's a good point, I think it will work better with that paragraph at the top 👍